.github/workflows/test.yaml

```diff
@@ -27,6 +27,7 @@ jobs:
                 php-version:
                     - '7.3'
                     - '7.4'
+                    - '8.0'
```

Comment:
You allow need to remove the part 
```
name: Configuration required for PHP 8.0
```
to be sure it's compatible with php8

Reply:
I missed it. Done.